.changeset/red-chefs-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/styled-base': patch
```

Comment:
this probably should include `@emotion/styled` package as well as they both are tightly coupled to each other